docs/host.md

```diff
@@ -49,70 +49,18 @@ This guide provides step-by-step instructions to install OpenVino&trade; Model S
    cd dist/centos && tar -xzvf ovms.tar.gz
    ```
 
-## Start the Serving<a name="start-the-serving"></a>
-1. The server can be started using the command : 
+## Running the Serving
+1. The server can be started using the command in the folder, where OVMS was installed: 
 ```Bash
-/ovms/bin/ovms -help
+./ovms/bin/ovms -help
```

Comment:
I think there should be `--help` here:
```bash
$ docker run -ti --rm openvino/model_server -help
error parsing options: Option ‘e’ does not exist
```

Reply:
use double dash --help
docker run -ti --rm openvino/model_server --help

Reply:
Yes. Use double dash. In document.